app/helpers/distribution_helper.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module DistributionHelper
+  VALUE = '_'
```

Comment:
? Perhaps add a comment? I'm guessing this is like an anonymous variable?

Reply:
This is just the default value for the structure. Imagine the structure not being a hash but just a single value, it would be this value. The reason that everything has to be a hash is so that we can nest combined scores into combined histograms that are constructed recursively. In order for that to work, here we choose a value that is short (because everything is stored in JSON format) and is probably not taken by any of the question IDs.

Reply:
Added the above comment to the code.